### DISCUSSION TITLE
他のコンテンツと重複する記述をどうするかの方向性

### DISCUSSION THREAD

**Discussion by rinotc** [scala-text/scala_text#866] [General]
#853 
で議論になった件。

https://github.com/scala-text/scala_text/pull/853#issuecomment-1061280156

> そもそもsbtの日本語公式とほぼ同じこと書くなら、わざわざ書く必要ある？
> という住み分けの問題は…？ 🤔

**Comment by rinotc** [comment]
想定読者層が

* 大学の情報学部卒である
* Java言語の基本知識がある
* 何か意味のあるアプリケーションを作ったことがある
* 趣味でTwitter APIなどを触ったり、プログラミングを行っている

であるならば、被ってるところはリンク貼ってあげればいい気もしますね。

https://github.com/scala-text/scala_text/pull/853#issuecomment-1061317679

でも述べられているように、他の話題に注力した方がいい気がします。

一応、#556 の野望があると思いますが、これは全くのプログラミング未経験者を想定しているわけではないので、それを想定してもツールのインストールくらいは適切にリンクを貼ってあげれば十分対応できると思います。

**Comment by xuwei-k** [comment]
そもそも、想定読者層自体も、あくまでも当初そうだっただけなんで、必要におうじて、多少変えるのもあり…？
すごく変えたいわけでもないというか、そんなに想定読者層どうするか?の意見も強く持ってないというか、まとまってないけど

**Reply by rinotc** [reply]
元々は研修テキストなので、研修で利用することを想定しつつ、どこまで経験が少ない人を想定するのかを検討すると変にブレなくていい気がします。

研修で利用されることを想定するなら、元々話題のsbt install とかは、会社のメンターがサポートすればいいのでその辺りの記述は公式 docへのリンクで済むのかなと思ったりします。

**Reply by kmizu** [reply]
@rinotc その辺棚上げになってたんでissue立ててくださったのは助かります。個人的にはもうちょい初学者向けにすると、国内向けの日本語書籍が今後出る見込みとか少なそうな中でちょうどよいのかなーと考えてます。具体的には、Javaを履修済みって前提なくしても読めるようにすると入ってきやすいのかなと（実際、スクリプト言語、特にRubyやPHPから入って来るケースなどがむしろ目立つ気もします）。

で、もしJava履修済みってのを外すなら、テキストのどの部分を改変すべきかってのは読み合わせが必要そうで、今後のテキスト修正の前に読み合わせ会みたいなのをすると意思統一出来て良さそうです。

**Comment by xuwei-k** [comment]
- Javaのインストール方法書いたことにより関連する話題 https://github.com/scala-text/scala_text/issues/995
